devito/passes/clusters/blocking.py

```diff
@@ -196,6 +196,10 @@ def callback(self, clusters, prefix):
         if self._has_short_trip_count(d):
             return clusters
 
+        # Pointless if there's no data reuse
+        if all(not self._has_data_reuse(c) for c in clusters):
```

Comment:
nit: Slight preference for but not important

```suggestion
        if not any(self._has_data_reuse(c) for c in clusters):
```

Reply:
agree, I will change it in a subsequent branch